### PR TITLE
Run rustfmt on CI

### PR DESCRIPTION
**Pull request by sgrif**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -48,6 +48,12 @@ matrix:
     - (cd diesel_codegen && cargo rustc --no-default-features --features "lint dotenv sqlite postgres mysql" -- -Zno-trans)
     - (cd diesel_infer_schema && cargo rustc --no-default-features --features "lint dotenv sqlite postgres mysql" -- -Zno-trans)
     - (cd diesel_compile_tests && travis-cargo test)
+  include:
+  - rust: nightly
+    env: RUSTFMT=YESPLEASE
```
Comment (Eijebong) [inline .travis.yml]: Why do we need this ?

Comment (sgrif) [reply .travis.yml]: To make it clear what this run is.

Comment (Eijebong) [reply .travis.yml]: Oh in the travis UI, makes sense 👍

Comment (sgrif) [reply .travis.yml]: Well we also need some env var that is different to make it a different matrix item than the normal nightly test runs.

**Pull request merged**